plugins/polio/js/src/constants/translations/en.json

```diff
@@ -93,7 +93,7 @@
     "iaso.polio.form.label.ORPGReview": "ORPG review",
     "iaso.polio.form.label.outbreakdeclarationdate": "Outbreak declaration date",
     "iaso.polio.form.label.override": "Override",
-    "iaso.polio.form.label.percentageCoveredTargetPopulationRound": "Percentage covered target population Round",
+    "iaso.polio.form.label.percentageCoveredTargetPopulationRound": "Percentage covered target population",
```

Comment:
Since it's not used anywhere else, you can change the key to  `iaso.polio.form.label.percentageCoveredTargetPopulation`